### PR TITLE
Renamed to LibreRambox

### PR DESCRIPTION
**Pull request by KevinRoebert**

### CODE REVIEW

**Comment by KevinRoebert** [comment]
Does anyone want to review this PR?

**Comment by fslori** [comment]
I didn't check for completeness, but looks good!

Nice that the installation packages are built as artifacts in the checks; I could install the debian package successfully.

Some related ideas:

- perhaps a dev version suffix can be added to the release version number right after each release, e.g. `x.y.z` becoming `x.y.z.<somenumber>`, in order to distinguish dev versions from released versions.
- should the repo still be called 'community-edition'? That might look like there is also an alternative in the LibreRambox space.

Probably more important: I suggest to state in the Readme that LibreRambox originated as a fork from Rambox CE 0.8.0 (if I'm right), and continues its version numbering (if you choose so).

**Comment by Neustradamus** [comment]
Dear @KevinRoebert,

Can you reply about all my tickets?

Thanks in advance.

**Comment by mdouchement** [comment]
LGTM

I still see this folder `~/Library/Application Support/LibreRambox/Partitions/rambox` on my macbook. It contains all storage/cache folder of Rambox.

File: package.json [hunk 1]
```diff
@@ -216,7 +211,7 @@
 		"electron-is-dev": "^2.0.0",
 		"electron-store": "^8.1.0",
 		"electron-updater": "^6.1.4",
-		"is-online": "^9.0.1",
+		"is-online": "^10.0.0",
```
Comment (mdouchement) [inline package.json]: This one cannot be updated due to the ESM way of import.
By the way, that's the only change between v9 and v10 as far as I remember.

**Pull request merged**